src/buildApplication/generateFunctionsMap.ts

```diff
@@ -436,3 +433,25 @@ type LooseNode = Node & {
   */
 	value: any;
 };
+
+function getChunkIdentifier(chunkKey: number): string {
+	return `__chunk_${chunkKey}`;
+}
+
+function getChunkImportFn(functionPath: string): (chunkKey: number) => string {
+	const functionNestingLevel = getFunctionNestingLevel(functionPath);
+	const accountForNestingPath = `../`.repeat(functionNestingLevel);
```

Comment:
I'd maybe do something like `path.join(...(new Array(functionNestingLevel).fill('..')))` so you don't need to specify the separator

Reply:
Nice! 🤩👍